src/fXmlRpc/Transport/CurlTransport.php

```diff
@@ -55,6 +55,10 @@ public function __destruct()
         }
     }
 
+     public function setCurlOpt($option,$value){
```

Comment:
The brace should go to the next line, also {{$option,$value}} should be {{$option, $value}}. What about just calling the method setOption?